console/package.json

```diff
@@ -47,7 +47,8 @@
 		"react": "^16.8.6",
 		"react-dom": "^16.8.6",
 		"roslib": "^0.20.0",
-		"socket.io-client": "^2.1.1"
+		"socket.io-client": "^2.1.1",
+		"webrtc-ips": "^0.1.4"
```

Comment:
Thanks for adding missing dependency.

Reply:
Actually, this is not a dependency and should be removed.

Reply:
Without this installed, an error complains it's missing. What package _does_ have this dependence aand is missing it? How do we update **that**?

Reply:
I have removed the dependency from `console/package.json` as it is no longer giving error for loading the console application. I had to remove `node_modules`  folder from `component` and `console` application and did `npm install && enact link` for both the folders.